### PR TITLE
[HSEARCH] 7.1.0.Alpha2

### PR DESCRIPTION
**Pull request by marko-bekhta**

### CODE REVIEW

File: search/releases/7.1/index.adoc [hunk 1]
```diff
@@ -58,3 +68,61 @@ List<Book> hits = searchSession.search( Book.class )
     .filter( f.match().field( "authors.firstName" ).matching( "arthur" ) )
 ).fetchHits( 20 );
 ----
+
+By its nature a `knn` predicate will always try to find nearest vectors,
```
Comment (yrodiere) [inline search/releases/7.1/index.adoc]:
```suggestion
By nature a `knn` predicate will always try to find `k` nearest vectors,
```

File: search/releases/7.1/index.adoc [hunk 2]
```diff
@@ -58,3 +68,61 @@ List<Book> hits = searchSession.search( Book.class )
...
+float[] coverImageEmbeddingsVector = /*...*/
+
+List<Book> hits = searchSession.search( Book.class )
+    .where( f ->
+        // Create a knn predicate as usual:
+        f.knn( 5 ).field( "coverImageEmbeddings" ).matching( coverImageEmbeddingsVector )
+        // Specify the required minimum similarity value, to filter out irrelevant results:
+        .requiredMinimumSimilarity( 5 ) )
+    .fetchHits( 20 );
+----
```
Comment (yrodiere) [inline search/releases/7.1/index.adoc]: I wouldn't mention this in the "What's new" section on the website, as this looks like a relatively advanced feature to me? I'd only mention it in the blog post.
YMMV.

**Review by yrodiere — COMMENTED** [review]
Thanks, just a few suggestions.

File: search/releases/7.1/index.adoc (hunk 2 above)
Comment (marko-bekhta) [reply search/releases/7.1/index.adoc]: That makes sense 😃 I'll remove this one, thanks!

**Pull request merged**